tests/e2e-ui-tests/page-objects/portal/workorder/new.po.js

```diff
@@ -0,0 +1,172 @@
+var consts = require('../../utils/constants');
+var utils = require('../../utils');
+
+var NewWorkorderPage = function() {
+  var workorderFormSelector = 'form[name="workorderForm"]';
+  var locators = {
+    workorderForm: {
+      self: element(by.css(workorderFormSelector)),
+      fields: {
+        title: element(by.css(workorderFormSelector + ' #inputTitle')),
+        address: element(by.css(workorderFormSelector + ' #inputAddress')),
+        latitude: element(by.css(workorderFormSelector + ' input[name="lattitude"]')),
+        longitude: element(by.css(workorderFormSelector + ' input[name="longitude"]')), // ID is wrong in UI
+        summary: element(by.css(workorderFormSelector + ' #inputSummary')),
+      },
+      dropdowns: {
+        assignee: element(by.css(workorderFormSelector + ' #assignee')),
+        workflow: element(by.css(workorderFormSelector + ' #workflow'))
+      },
+      datetime: {
+        // startDate: element(by.css(workorderFormSelector + ' #inputStartDate')), // TODO
+        // startTime: element(by.css(workorderFormSelector + ' #inputStartTime')), // TODO
+        finishDate: element(by.css(workorderFormSelector + ' #inputFinishDate')),
+        finishTime: element(by.css(workorderFormSelector + ' #inputFinishTime')),
+      },
+      warnings: {
+        workflow: element(by.css(workorderFormSelector + ' [ng-messages="workorderForm.workflow.$error"] div')),
+        title: element(by.css(workorderFormSelector + ' [ng-messages="workorderForm.title.$error"] div')),
+        address: element(by.css(workorderFormSelector + ' [ng-messages="workorderForm.address.$error"] div')),
+        latitude: element(by.css(workorderFormSelector + ' [ng-messages="workorderForm.lattitude.$error"] div')),
+        longitude: element(by.css(workorderFormSelector + ' [ng-messages="workorderForm.longitude.$error"] div')),
+        // startDate: element(by.css(workorderFormSelector + ' [ng-messages="workorderForm.startDate.$error"] div')), // TODO
+        // startTime: element(by.css(workorderFormSelector + ' [ng-messages="workorderForm.startTime.$error"] div')), // TODO
+        finishDate: element(by.css(workorderFormSelector + ' [ng-messages="workorderForm.finishDate.$error"] div')),
```

Comment:
No longer available - let's merge it this way and we can adjust that later. Just letting you know.